projects/storefrontlib/src/cms-components/myaccount/order/order-return-requests/order-return-requests.component.html

```diff
@@ -0,0 +1,118 @@
+<ng-container *ngIf="returnRequests$ | async as returnRequests">
```

Comment:
This is a global question, so `order-return-requests` is the list. Can't it be call `order-return-request-list`? 

Also, should we add a list item component like we have in cart and other lists

Reply:
yes!, I will rename them.